fees/chainlink.ts

```diff
@@ -69,6 +77,8 @@ const gasTokenId: IGasTokenId = {
   [CHAIN.ETHEREUM]: "coingecko:ethereum",
   [CHAIN.BSC]: "coingecko:binancecoin",
   [CHAIN.POLYGON]: "coingecko:matic-network",
+  [CHAIN.FANTOM]: "coingecko:fantom",
+  [CHAIN.AVAX]: "coingecko:avalanche-2",
 }
 const _providers: TProvider =  {
```

Comment:
do the current providers not work?

Reply:
ohh let me check, i think should switch to use getProivder from our sdk 🙏